lib/IRGen/IRGenSIL.cpp

```diff
@@ -5245,6 +5245,16 @@ void IRGenSILFunction::visitWitnessMethodInst(swift::WitnessMethodInst *i) {
 
   auto fnType = i->getType().castTo<SILFunctionType>();
 
+  if (IGM.isResilient(conformance.getRequirement(),
+                      ResilienceExpansion::Maximal)) {
+    auto *fnPtr = IGM.getAddrOfDispatchThunk(member, NotForDefinition);
+    auto sig = IGM.getSignature(fnType);
+    FunctionPointer fn(fnPtr, sig);
```

Comment:
Please use FunctionPointer::forDirect here, since you can.

Reply:
Done